omod/src/main/webapp/pages/conditionlist/manageConditions.gsp

```diff
@@ -40,7 +41,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
 <h2>${ui.message("coreapps.conditionui.manageConditions")}</h2>
 
 <div id="condition" ng-app="conditionsApp" ng-controller="ConditionsController"
-     ng-init="conditions = getConditions('${patient.uuid}')">
+     ng-init="conditions = getConditions('${patient.uuid}'); config = { dateFormat: '${ ui.getJSDateFormat() }', locale: '${ ui.getLocale().getLanguage() }'};">
     <div id="tabs">
```

Comment:
This is setting things up on the page so that the angular components have what they need from the UI framework (moment.js with localization and the configured date format and locale from the server) to support localized date formats.